### PR TITLE
add new empty op (PR #26659) doc

### PR DESCRIPTION
**Pull request by windstamp**
add new empty op doc (Chinese version)

### CODE REVIEW

**Comment by windstamp** [comment]
image

**Comment by windstamp** [comment]
image

File: doc/paddle/api/paddle/tensor/creation/empty_cn.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+.. py:function:: paddle.empty(shape, dtype=None, name=None)
+
+
+
+该OP创建形状大小为shape并且数据类型为dtype的Tensor，其中元素值是未初始化的。
+
+参数：
+    - **shape** (list|tuple|Tensor) – 指定创建Tensor的形状(shape), 数据类型为int32 或者int64。
+    - **dtype** （np.dtype|core.VarDesc.VarType|str， 可选）- 输出变量的数据类型。若为None，则输出变量的数据类型为系统全局默认类型，默认值为None。
```
Comment (jzhang533) [inline doc/paddle/api/paddle/tensor/creation/empty_cn.rst]: no core,VarDesc.VarType
英文文档里有支持了哪些数据类型，这里没有。

File: doc/paddle/api/paddle/tensor/creation/empty_cn.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - **name** （str，可选）- 具体用法请参见 :ref:`api_guide_Name`，一般无需设置，默认值为None。
+    
+返回：返回一个存储结果的Tensor，数据类型和dtype相同。
```
Comment (jzhang533) [inline doc/paddle/api/paddle/tensor/creation/empty_cn.rst]: 英文是：
        Tensor: Tensor which is created according to ``shape`` and ``dtype``, and is uninitialized.
跟英文不一致。

File: doc/paddle/api/paddle/tensor/creation/empty_cn.rst (hunk 1 above)
Comment (windstamp) [reply doc/paddle/api/paddle/tensor/creation/empty_cn.rst]: thanks。已修改

File: doc/paddle/api/paddle/tensor/creation/empty_cn.rst (hunk 2 above)
Comment (windstamp) [reply doc/paddle/api/paddle/tensor/creation/empty_cn.rst]: thanks。已修改

**Comment by windstamp** [comment]
image

**Review by jzhang533 — APPROVED** [review]
lgtm

**Pull request merged**